apps/hash-api/src/integrations/sync-back-watcher.ts

```diff
@@ -40,38 +40,43 @@ export const createIntegrationSyncBackWatcher = async (
   const queue = new RedisQueueExclusiveConsumer(redisClient);
 
   const processQueueMessage = () => {
-    void queue.pop(queueName, null, async (item: string) => {
-      const message = JSON.parse(item) as Wal2JsonMsg;
+    queue
```

Comment:
Strange, I would've expected eslint to complain about `void` missing here. Is it not required when using `.catch`?


```suggestion
    void queue
```

Reply:
Correct - the requirements are that one of the following list applies:
> Promises must be awaited, end with a call to .catch, end with a call to .then with a rejection handler or be explicitly marked as ignored with the `void` operator